### PR TITLE
New attempt to make e2e nodes work without bootstrap.py

### PR DESCRIPTION
**Pull request by rikatz**
Signed-off-by: Ricardo Pchevuzinske Katz <ricardo.katz@gmail.com>

Hopefully this is my last attempt to make this job work without bootstrap.py

I'm sorry for spamming the repo this way :/

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -52,10 +52,17 @@ presubmits:
     always_run: false
     optional: true
     decorate: true
+    extra_refs:
+    - base_ref: master
+      path_alias: k8s.io/kubernetes
```
Comment (chaodaiG) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:57]: We don't need these, instead:
```suggestion
    path_alias: k8s.io/kubernetes
```

Comment (rikatz) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml]: Changed here :)

**Review by rikatz — COMMENTED** [review]
Changed

**Pull request merged**